swagger/src/main/scala/org/http4s/rho/swagger/models.scala

```diff
@@ -187,6 +187,15 @@ object models {
     , vendorExtensions : Map[String, Any]  = Map.empty
     ) {
 
+    def operations: List[Operation] =
```

Comment:
No problem, I can change it. I have taken the habit to be a lot more specific on my collection return types now because I think that it is better to indicate that the collection I am returning will not be infinite for example. But I am fine with changing it.